### PR TITLE
Add Dependencies list pass through to include_router

### PR DESCRIPTION
**Pull request by alexrejto**

### CODE REVIEW

**Comment by dsinghvi** [comment]
@alexrejto we just added a [contributing.md](https://github.com/fern-api/fern-python/blob/main/CONTRIBUTING.md) to this repo, mind updating the PR by running the snapshot tests? 

`poetry run pytest tests/fastapi --snapshot-update`

File: src/fern_python/external_dependencies/fastapi.py [hunk 1]
```diff
@@ -129,11 +129,11 @@ def Query(
         )
 
     @staticmethod
-    def include_router(*, app_variable: str, router: AST.Expression) -> AST.Expression:
+    def include_router(*, app_variable: str, router: AST.Expression, args: list[AST.Expression] = []) -> AST.Expression:
```
Comment (dsinghvi) [inline src/fern_python/external_dependencies/fastapi.py:132]: @alexrejto can you use `typing.List` ? This generator is on python 3.8 where `list` is not available

**Comment by GfxKai** [comment]
Need any help moving this forwards? I'm trying to use Fern with [fastapi-auth0](https://github.com/dorinclisu/fastapi-auth0) and desperately need this functionality!

Looks like tests are failing due to mypy complaining about the type of the param:
```py
dependencies: typing.List[fastapi.Depends]
```

> register.py:40: error: Function "fastapi.param_functions.Depends" is not valid as a type  [valid-type]
> register.py:40: note: Perhaps you need "Callable[...]" or a callback protocol?

**Comment by dsinghvi** [comment]
@GfxKai  feel free to copy the branch and keep moving it forward, will merge it in once its green!

**Pull request closed without merging**